api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -3977,4 +3982,99 @@ public void saveOrder_shouldSaveTheFormNamespaceAndPath() {
 		order = orderService.saveOrder(order, null);
 		assertEquals(NAMESPACE + "^" + FORMFIELD_PATH, order.getFormNamespaceAndPath());
 	}
+
+	@Test
+	public void getAllOrderAttributeTypes_shouldReturnAllOrderAttributeTypes() {
+		assertEquals(4, orderService.getAllOrderAttributeTypes().size());
+	}
+
+	@Test
+	public void getOrderAttributeTypeById_shouldReturnNullIfNoOrderAttributeTypeHasTheProvidedId() {
+		assertNull(orderService.getOrderAttributeTypeById(15));
+	}
+
+	@Test
+	public void getOrderAttributeTypeById_shouldReturnOrderAttributeTypeUsingProvidedId() {
+		OrderAttributeType orderAttributeType = orderService.getOrderAttributeTypeById(2);
+		assertThat(orderAttributeType.getId(), is(2));
+	}
+
+	@Test
+	public void getOrderAttributeTypeByUuid_shouldReturnOrderAttributeTypeUsingProvidedUuid() {
+		OrderAttributeType orderAttributeType = orderService.getOrderAttributeTypeByUuid(
```

Comment:
Is it possible to have this as 
`assertEquals("Referral", orderService.getOrderAttributeTypeByUuid("9758d106-79b0-4f45-8d8c-ae8b3f25d72a").getName());`
eliminating line 4004